account_einvoice_generate/models/account_move.py

```diff
@@ -23,7 +23,7 @@ def _xml_format_in_pdf_invoice(self):
         if (
             xml_format
             and xml_format != "none"
-            and self.move_type in ("out_invoice", "out_refund")
+            and self.move_type in self._select_moves_type_for_xml_in_pdf()
```

Comment:
```suggestion
            and self.move_type in self._select_move_type_for_xml_in_pdf()
```

To be more constistent with the field

Reply:
Thanks, done